doc/contributing.md

```diff
@@ -32,40 +36,24 @@ clj -M:ffix
 
 ## Releasing Datahike
 ### Deploying Datahike to Clojars
-```
-clj -M:deploy
-```
+First you have to build the artifact with `clj -T:build jar`, then you need to
+set `CLOJARS_USERNAME` and `CLOJARS_PASSWORD` as environment variables first,
+then you can run `clj -T:build deploy` to deploy the artifact to Clojars.
 
-### SNAPSHOT
-To create a snapshot of Datahike and upload it to Clojars the version in
-`pom.xml` needs to end on *-SNAPSHOT*. Snapshots can and will be overwritten
-if a snapshot with the current version is already present. There is no easy way
-to recognize that there is a new snapshot available on Clojars.
-
-A snapshot in Datahike is created by committing/merging on and pushing to the
-*development* branch. Then the snapshot gets validated. This means the version
-string from `pom.xml` is extracted and it needs to end on *-SNAPSHOT* and
-the current branch needs to be *development*. Only then a snapshot gets released
-to Clojars.
-
-### Release
-To create a release of Datahike and upload it to Clojars the version in `pom.xml`
-needs to be incremented at the respective position. Then you can commit and push the
-current version to GitHub. CircleCI will then run unit and integration tests and build
-this version of Datahike. When this succeeds it releases the new version to Clojars
-only if the version string does *not* end on *-SNAPSHOT* and the current branch is
-*master*.
-
-The deployment to Clojars will fail if this version is already present on Clojars. It is
-not possible to overwrite a release, only snapshots can be overwritten.
+Datahike is released everytime there is a commit to the `main` branch. We are
+using semantic versioning and the patch-version is the number of commits on
+the main branch. That means it is always increasing, even if the major or minor
+version is increased.
 
 ### CircleCI
-On CircleCI unit and integration tests are run. Afterwards, Datahike is being compiled and
-released. The official CircleCI jdk-8 image is used and the secret key to release to
 Clojars needs to be stored as an environment variable on CircleCI. There needs to be a
 variable `CLOJARS_USERNAME` set to your Clojars username and a variable `CLOJARS_PASSWORD` set
 to the token that permits to deploy on clojars.
 
+### Git tags
+Each release gets a release entry in Github and a git tag to point to the merge commit
+made when merging a branch into `main`. The jar is appended to the Github-release.
```

Comment:
How would the process with Gtihub releases work? Is a draft created automatically or is this done by the pipeline? And is it possible to fill in the text in the release text?

Reply:
There is a github-draft-release made by the release-step of the pipeline. We can fill the text in the release as well with e.g. the CHANGELOG.md though this is huge.

Reply:
We could make it a non-draft-release though...What do you prefer?

Reply:
We will for now manually add release-notes to the release. Thinking about automating this soonish.